PHP: add more-examples link (#3712)
open-telemetry/opentelemetry.io: content/en/docs/instrumentation/php/automatic.md

After you have automatic instrumentation configured for your app or service, you
might want to add [manual instrumentation](../manual) to collect custom
telemetry data.

> ## More examples
> You can find more examples and documents here [PHP examples](https://github.com/open-telemetry/opentelemetry-php-contrib)

Reviewer: ```suggestion
## Examples

You can find more examples in the [opentelemetry-php-contrib](https://github.com/open-telemetry/opentelemetry-php-contrib) repository.
```

After you have automatic instrumentation configured for your app or service, you
might want to add [manual instrumentation](../manual) to collect custom
telemetry data.

> 
> 
> ## Examples
> 
> You can find more examples in the [opentelemetry-php-contrib](https://github.com/open-telemetry/opentelemetry-php-contrib) repository.

Reviewer 2: Hmm, it doesn't make sense to have a section head after "Next steps" IMHO, how about just adding the sentence to the existing last section, something like this:

```suggestion

For more examples, see
[opentelemetry-php-contrib/examples](https://github.com/open-telemetry/opentelemetry-php-contrib/tree/main/examples).
```
